Jenkinsfile

```diff
@@ -104,7 +114,8 @@ if( buildParams.BUILD_OS != "all" ){
 
 // Using .each{} hits "a CPS-transformed closure is not yet supported (JENKINS-26481)"
 for( label in build_nodes) {
-  stage_unit_test(label)
-  stage_rpmbuild(label)
-  stage_integration(label)
+//stage_unit_test(label)
+//stage_rpmbuild(label)
+//stage_integration(label)
+  stage_clean_rpms(label)
 }
```

Comment:
Please revert the changes to ``Jenkinsfile``. This causes the existing build process halts and cleanup tasks is performed by another Jenkins job not by ``citest`` job.